### PR TITLE
feature: use path to search

### PR DESCRIPTION
**Pull request by 847850277**
Hi community: 
    I have completed the feature use path to search.
    please review the code.
    see issues [use path to search](https://github.com/apache/incubator-shenyu/issues/3631)

### CODE REVIEW

**Comment by dragon-zhang** [comment]
good job, I will check it.







------------------ Original ------------------
From: zhengpeng ***@***.***&gt;
Date: Thu,Jun 30,2022 5:19 PM
To: apache/incubator-shenyu-dashboard ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [apache/incubator-shenyu-dashboard] feature: use path to search (PR #213)




Hi community:
 I have completed the feature use path to search.
 please review the code.
 see issues use path to search
 
 
You can view, comment on, or merge this pull request online at:
 
&nbsp; https://github.com/apache/incubator-shenyu-dashboard/pull/213
 
Commit Summary
    
007e70f  feature: 
 
File Changes 
 
(2 files)
    
     M     src/routes/System/Metadata/index.js     (13)      

     M     src/services/api.js     (4)    
 
Patch Links:
    
https://github.com/apache/incubator-shenyu-dashboard/pull/213.patch   

https://github.com/apache/incubator-shenyu-dashboard/pull/213.diff

File: src/routes/System/Metadata/index.js [hunk 1]
```diff
@@ -398,9 +399,9 @@ export default class Metadata extends Component {
       <div className="plug-content-wrap">
         <div style={{ display: "flex" }}>
           <Input
-            value={appName}
+            value={path}
             onChange={this.searchOnchange}
-            placeholder={getIntlContent("SHENYU.META.INPUTAPPNAME")}
+            placeholder={getIntlContent("SHENYU.META.PATH")}
```
Comment (dragon-zhang) [inline src/routes/System/Metadata/index.js:404]: add and replace to `placeholder={getIntlContent("SHENYU.META.INPUTPATH")}`

Comment (847850277) [reply src/routes/System/Metadata/index.js:404]: when i use "INPUTPATH", the placeholder has not default value. 
image

and i use "PATH",the placeholder has default value. 

image

Comment (dragon-zhang) [reply src/routes/System/Metadata/index.js:404]: I mean add `SHENYU.META.INPUTPATH` to `zh-CN.json` and `en-US.json`.

Comment (847850277) [reply src/routes/System/Metadata/index.js]: Thank you. I see. I have modified this place.

**Review by dragon-zhang — APPROVED** [review]

**Comment by dragon-zhang** [comment]
@847850277 happy contribution ! please add wechat Z18179469818

**Pull request merged**